### ISSUE TITLE
[SF][UI5-Input][BUG][Urgent][0.31.22]  Space can not be entered into ui5-input if ui5-input is wrapped with ui5-table-row component 

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#4365]
Dear colleagues,
I am from SuccessFactors Web Experience Shanghai team.

The version of ui5-webcomponent is **0.31.22**

**Issue:** 
Space can not be entered into ui5-input if ui5-input is wrapped with ui5-table-row component 

**Expected Behavior:**
Space can be entered into ui5-input if ui5-input is wrapped with ui5-table-row component 

The below code in [ui5-table-row](https://github.com/SAP/ui5-webcomponents/blob/release-0.31/packages/main/src/TableRow.js#L109) caused this issue

```
_onkeydown(event) {
		if (isEnter(event)) {
			this.fireEvent("row-click", { row: this });
		}

		if (isSpace(event)) {
			event.preventDefault();
		}
	}
```

**Comment by IlianaB** [comment]
Hello @SAP/ui5-webcomponents-topic-rl  I am forwarding this issue to you, because in ui5-webcomponent version 0.31.22 Space can not be entered into ui5-input, if ui5-input is wrapped with ui5-table-row component: https://codesandbox.io/s/ui5-webcomponents-forked-2ynvz?file=/index.html

It seems to already be fixed in 0.33.

**Comment by d3xter666** [comment]
Hi @sfsf-xweb-sh ,

This is a duplicate with: https://github.com/SAP/ui5-webcomponents/issues/3390
It's already fixed with **v0.33.0**

Cheers

**Comment by sfsf-xweb-sh** [comment]
Hi @IlianaB  @d3xter666 
We must use SF version of UI5-WebComponent.
Can you downport this fix into 0.31.x version ?
Thanks

**Comment by niyap** [comment]
Hello @sfsf-xweb-sh,

The fix is downported in 0.31 with the following commit: https://github.com/SAP/ui5-webcomponents/commit/f891e9d19658c4a8949ff003ede9edc4156d445f

Kind Regards,
Niya
